docs/api/commands/exec.mdx

```diff
@@ -200,8 +201,8 @@ describe('has data available from database', { execTimeout: 90000 }, () => {
 
 ### Assertions [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Assertions)
 
-- `cy.exec()` will only run assertions you have chained once, and will not
-  [retry](/guides/core-concepts/retry-ability).
+- `cy.exec()` asserts that the command exits successfully. If the command exits
+  with a non-zero exit code, `cy.exec()` will fail the test.
```

Comment:
if this is the case, why does it yield stderr? Users can assert of use that value?

Reply:
Many commands write to stderr even when successful. `curl` is one example.

https://askubuntu.com/questions/944788/how-does-curl-print-to-terminal-while-piping

> When you are using curl to open an URL, you'll get two output:
>
>     The status of the curl itself.
>     The contents of that URL.
>
> Curl should use a way to show these two separately otherwise processing of the real output (URL's content) would be hard and I'll end up with unnecessary contents (curl's status).
>
> So it uses stderr for its status and stdout for the content.

You wouldn't usually `curl` with `cy.exec`, but it's a common pattern in linux commands.